scripts/deploy_from_template.sh

```diff
@@ -37,4 +37,13 @@ fi
 oc process -p IMAGE=$IMAGE -p IMAGE_TAG=$TAG -f template.yaml --local | oc apply -n $NAMESPACE -f -
 
 sleep 5
-oc rollout status  -n $NAMESPACE deployment/assisted-service-mcp --timeout=300s
\ No newline at end of file
+if  ! oc rollout status  -n $NAMESPACE deployment/assisted-service-mcp --timeout=300s; then
+    echo "Deploying assisted-chat failed"
+    ASSISTED_SERVICE_MCP_POD=$(oc get pods -n "$NAMESPACE" | tr -s ' ' | cut -d ' ' -f1 | grep assisted-service-mcp)
```

Comment:
The bot has a point at least about this line. You can use a selector for this rather than parsing out the pod name.

Something like this would work even for multiple replicas: `oc logs -l app=assisted-service-mcp --timestamps --prefix --tail=-1`